metadata-ingestion/src/datahub/ingestion/source/looker/lookml_source.py

```diff
@@ -1501,8 +1501,7 @@ def get_workunits_internal(self) -> Iterable[MetadataWorkUnit]:
                         p_ref = p_checkout_dir.resolve()
                     except Exception as e:
                         logger.warning(
-                            f"Failed to clone remote project {project}. This can lead to failures in parsing lookml files later on",
-                            e,
+                            f"Failed to clone remote project {project}. This can lead to failures in parsing lookml files later on: {e}",
```

Comment:
I think you can do `logger.warning(..., exc_info=True)` if you want? Although that includes a stack trace

Reply:
Yeah I explicitly want the error message but not the full stack trace